report/src/future_work/future_work.tex

```diff
@@ -0,0 +1,21 @@
+\chapter{Future work}
+As mentioned in Section \ref{sec:fingerprinting}, the number of beacons and their placement affects the quality of the localization results when using fingerprinting. 
+Similarly, the physical layout and the obstacles within a room can affect the classification performed in the second phase of fingerprinting. 
+It is therefore relevant to experiment with several configurations related to beacon placement, number of beacons, physical room layout, and different radio wave noise within the room and in the surrounding environment. 
+
+Future research pursuits within room occupancy detection research, leveraging fingerprinting using BLE, should prioritize the discovery of a configuration that shows improved generalization across diverse settings. 
+One approach to achieving this involves further studies of strategies that can improve either the generated map or the classification algorithm.
+Our experiments showed that the quality of classifications degraded for positions outside the room by up to $49.91$ percent compared to indoor positions, as outlined in Table \ref{lst:resultsPhone_precision}. Therefore, future research should experiment with different beacon configurations where, for example, one or more beacons are placed outside the meeting room. 
+Future experiments might show an improvement in the map quality resulting in better classification. Alternatively, it could show that fewer beacons are needed to achieve similar results. 
+Further to this, an exploration of alternative algorithms in the pursuit of improved generalization would be beneficial.
```

Comment:
What is meant by generalization?

In my view, generalization would be for a trained model (which kNN isn't) to work with other data than it has seen during training.